src/osversion/VersionTranslator.cpp

```diff
@@ -6826,12 +6826,12 @@ namespace osversion {
 
   }  // end update_3_2_1_to_3_3_0
 
-  std::string VersionTranslator::update_3_3_0_to_3_3_1(const IdfFile& idf_3_3_0, const IddFileAndFactoryWrapper& idd_3_3_1) {
+  std::string VersionTranslator::update_3_3_0_to_3_4_0(const IdfFile& idf_3_3_0, const IddFileAndFactoryWrapper& idd_3_4_0) {
```

Comment:
Ok, no VT needed, all these fields were added at the end of the object (not inserted) and have a default, so no-op